common/src/main/java/io/github/miniplaceholders/expansion/luckperms/common/CommonExpansion.java

```diff
@@ -38,6 +38,15 @@ public Expansion.Builder commonBuilder() {
                     }
                     return Tag.inserting(parsePossibleLegacy(user.getCachedData().getMetaData().getSuffix()));
                 })
+                .audiencePlaceholder("get_meta", (aud, queue, ctx) -> {
```

Comment:
Could you change the name of the placeholder to simply "meta"? This would make it easier to migrate from the LuckPerms PlaceholderAPI expansion

Reply:
sure just updated it